bitnami/spring-cloud-dataflow/templates/NOTES.txt

```diff
@@ -94,7 +94,7 @@ To access Spring Cloud Data Flow dashboard from outside the cluster execute the
     {{- $requiredExternalDbPassword := dict "valueKey" "externalDatabase.password" "secret" $secretNameExternalDb "field" "mariadb-password" -}}
     {{- $passwordWarnings =  append $passwordWarnings $requiredExternalDbPassword -}}
   {{- end -}}
-  {{- if and (.Values.externalRabbitmq.enabled) (not .Values.rabbitmq.enabled) (not .Values.externalRabbitmq.existingPasswordSecret) -}}
```

Comment:
How are you deploying the chart? For this block to execute you not only need to set `rabbitmq.enabled` to false (as you did), but also `externalRabbitmq.enabled` to true, which does not seem to be your case.

Reply:
externalRabbitmq is false in the [default values.yaml](https://github.com/bitnami/charts/blob/master/bitnami/spring-cloud-dataflow/values.yaml#L974) of the spring-cloud-dataflow chart.

Reply:
If you are using the default false value then this line is not affecting you as it requires `.Values.externalRabbitmq.enabled` to be true.

Reply:
you are absolutely right! sorry. :(

Reply:
I'm going to close the PR. Let's try to identify the issue at #5781